src/deroff.rs

```diff
@@ -703,34 +703,31 @@ impl Deroffer {
     }
 
     fn request_or_macro(&mut self) -> bool {
+        // self.s[0] is a period or open single quote
         self.skip_char(1);
 
-        let s0 = self.s.chars().nth(1).unwrap_or('_'); // _ will be ignored by the match
-
-        match s0 {
-            '\\' => {
-                if self.str_at(1) == "\"" {
-                    self.condputs("\n");
-                    return true;
-                }
+        match self.s.chars().nth(1) {
+            Some('\\') if self.str_at(1) == "\"" => {
```

Comment:
This branch is unreachable:
```rust
self.s.chars().nth(1).to_string() == self.str_at(1)
```
I noticed this in my `test_deroff` branch. I don't know if it's a logic error on their end, or if im misunderstanding

Reply:
Haha, I just ported the exact code from the previous version, I didn't saw that this is unreachable. I guess I will just delete this branch.